install/kubernetes/helm/subcharts/tracing/values.yaml

```diff
@@ -22,7 +22,19 @@ jaeger:
       # - secretName: jaeger-tls
       #   hosts:
       #     - jaeger.local
-replicaCount: 1
+zipkin:
+  hub: docker.io/openzipkin/zipkin
```

Comment:
hub seems to be used in other sub-charts for the org, rather than the image - so maybe change to `docker.io/openzipkin` and add the `zipkin` image name into the deployment-zipkin.yaml.

Reply:
done.